templates/operations.html

```diff
@@ -1114,6 +1139,40 @@ <h3 x-text="selectedOperation.name"></h3>
             addNewRow: false,
             addedTechniquesIDs: new Set(),
 
+            searchText: '',
+            searchCriteria: 'decide',
+
+            get filterFromSelectedOperation() {
```

Comment:
Super nit-picky, but I'd rename this to `filterLinksFromSelectedOperation` only because there are a ton of functions on this page so it helps to be explicit.

Reply:
Done, and also changed variable names.